core/src/edu/asu/c3simulator/screens/MarketingWizard.java

```diff
@@ -0,0 +1,362 @@
+					skin);
+		
+		campaignTargetLabel.setDisabled(true);
+		campaignTargetLabel.setColor(0.5f, 0.5f, 0.5f, 1.0f);
+		
+		TextButton backButton = new TextButton("Back", skin);
+		backButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				game.setScreen(firstWizardPage);
+			}
+		});
+		
+		upperSection.add(backButton).center();
+		upperSection.add(campaignTargetLabel).expand().fill().center();
+		
+		Table campaignTypes = new Table();
+		Table campaignOptionTable = new Table();
+		Table productAndConsumerInformation = new Table();
+		
+		Label campaignTypesLabel = new Label("Campaign Types", skin);
+		List<String> campaignTypesList = new List<String>(skin);
+		campaignTypesList.setItems(getCampaignTypes());
+		campaignTypesList.addListener(new ChangeListener() {
+			
+			@Override
+			public void changed(ChangeEvent event, Actor actor)
+			{
+				marketingOption = campaignTypesList.getSelected();
+				updateCampaignOptions();
+			}
+			
+		});
+		ScrollPane campaignTypesScroll = new ScrollPane(campaignTypesList, skin);
+		
+		campaignTypes.add(campaignTypesLabel).center().row();
+		campaignTypes.add(campaignTypesScroll).expand().fill();
+		
+		campaignOptions = new Table();
+		marketingOption = campaignTypesList.getSelected();
+		Label campaignOptionsLabel = new Label("Campaign Options", skin);
+		ScrollPane campaignOptionsScroll = new ScrollPane(campaignOptions, skin);
+		updateCampaignOptions();
+		
+		campaignOptionTable.add(campaignOptionsLabel).center().row();
+		campaignOptionTable.add(campaignOptionsScroll).expand().fill();
+		
+		Label productAndConsumerInformationLabel1 = new Label("Product and", skin);
+		Label productAndConsumerInformationLabel2 = new Label("Consumer Information",
+				skin);
+		ScrollPane productAndConsumerInformationScroll = new ScrollPane(
+				getProductAndConsumerInformation(), skin);
+		TextButton finishButton = new TextButton("Finish", skin);
+		finishButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				// TODO add wizard results to data layer
+			}
+		});
+		
+		productAndConsumerInformation.add(productAndConsumerInformationLabel1).center()
+				.row();
+		productAndConsumerInformation.add(productAndConsumerInformationLabel2).center()
+				.row();
+		productAndConsumerInformation.add(productAndConsumerInformationScroll).expand()
+				.fill().row();
+		productAndConsumerInformation.add(finishButton).expandX().fillX().padTop(5.0f)
+				.padLeft(5.0f);
+		
+		lowerSection.add(campaignTypes).expand().fill();
+		lowerSection.add(campaignOptionTable).expand().fill();
+		lowerSection.add(productAndConsumerInformation).expand().fill();
+		
+		mainTable.add(campaignMethodLabel).left().row();
+		mainTable.add(upperSection).expandX().fillX().padBottom(10.0f).row();
+		mainTable.add(lowerSection).expand().fill();
```

Comment:
Nice names. I really get a sense of what this looks like, just by reading it. I ran the code to verify I was thinking the same thing it was creating - it is. Very nice.